composer.json

```diff
@@ -57,7 +58,9 @@
   "prefer-stable": true,
   "scripts": {
     "cs-check": "phpcs -p -s --standard=vendor/spryker/code-sniffer/Spryker/ruleset.xml src/ tests/",
```

Comment:
* Raise the supported PHP version to the `7.3`.
* Run the following console command in order to generate psalm report file for the module:
- `spryker-dev-console dev:analyze-psalm -m SprykerEco.Computop`.

Reply:
Done

Reply:
Raise the version of the "spryker-eco/computop-api" module to ^1.1.0.

Reply:
It trows error will do it before release.

Reply:
What's the error you received? I've checked, looks like working.

Reply:
I don't remember now it was few months ago but it was related to:
"Raise the version of the "spryker-eco/computop-api" module to ^1.1.0."